### ISSUE TITLE
A few more Math functions?

### ISSUE THREAD

**Issue by alexandervrs** [zrispo/wick#654]
I feel like we could have a few more Math functions for now that would help with animation, currently there seems to be only Math.abs()

Basically:

`
Math.sin(value);
Math.cos(value);
Math.tan(value);

Math.min(values, ... );
Math.max(values, ... );

Math.floor(value);
Math.ceil(value);
Math.round(value);

Math.pow(value, power);
Math.sqrt(value);

Math.sign(value);

Math.PI
`

**Comment by bbblitz** [comment]
I'm not certain why Math.abs() is in the documentation for wick, it seems like it's part of most [browser's javascript standard library](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Math)?

**Comment by zrispo** [comment]
Yeah I thought it might be helpful to have the most commonly used javascript functions in the docs for wick, but I don't think I ever finished them apparently:

![screen shot 2017-08-30 at 2 43 39 pm](https://user-images.githubusercontent.com/4970417/29889295-a56170bc-8d91-11e7-9020-919884b1f4b6.png)

I don't even know if people should have to use the standard javascript functions, maybe wick should have its own wrappers for the most common things? Any thoughts?
